database/pgx/pgx.go

```diff
@@ -479,7 +476,7 @@ func (p *Postgres) ensureVersionTable() (err error) {
 		return nil
 	}
 
-	query = `CREATE TABLE IF NOT EXISTS ` + p.quoteIdentifier(p.config.MigrationsTable) + ` (version bigint not null primary key, dirty boolean not null)`
+	query = `CREATE TABLE IF NOT EXISTS "` + p.config.migrationsSchemaName + `"."` + p.config.migrationsTableName + `" (version bigint not null primary key, dirty boolean not null)`
```

Comment:
Should we use `p.quoteIdentifier()` here?
Also, I think we can remove the `if p.config.MigrationsTableQuoted` check from `quoteIdentifier()` since it's also used to quote all identifiers, not just the migrations table name.

Reply:
> Also, I think we can remove the if p.config.MigrationsTableQuoted check from quoteIdentifier() since it's also used to quote all identifiers, not just the migrations table name.

@dhui Yes if I replace `p.quoteIdentifier(p.config.MigrationsTable)` by 

```
...` + quoteIdentifier(p.config.migrationsSchemaName) + `.` + quoteIdentifier(p.config.migrationsTableName) + `...
```

everywhere.

I'm going to apply this change 👍

Reply:
Done